### PR TITLE
Notifications endpoint

### PR DESCRIPTION
**Pull request by AnnaStepakova**
Add to Notification model date_created and receiver_user fields. Add uuid unique field to BaseUserProfile model (to make uuid a pk all migratins should be removed and migrated again, but I thought it's not the best idea as I have some handmade migrations, also some migrations were connected).

Add endpoint to notifications views and tests for it. 

Also fix issue with users serializers: fields birth_date and user were forgotten what caused an error (also I noticed there no tests for LandlordProfile)

### CODE REVIEW

File: notifications/views.py [hunk 1]
```diff
@@ -1,24 +1,12 @@
...
-        template.subject,
-        text,
-        'from@example.com',
-        ['to1@example.com'],
-        connection=connection,
-    )
-    email.send()
-    connection.close()
+    def get_queryset(self):
+        queryset = self.queryset.filter(receiver_user=self.kwargs['user_uuid'])
```
Comment (m0nk-tnd) [inline notifications/views.py]: можем ли получить параметр для фильтрации не из урл а из текущего пользователя?
Чтобы можно было получать только свои уведомления.

File: notifications/views.py [hunk 2]
```diff
@@ -1,24 +1,12 @@
-from django.template.loader import render_to_string
-from django.core import mail
+from rest_framework import generics
+from .models import Notification
+from .serializers import NotificationSerializer
 
-connection = mail.get_connection()
 
+class NotificationList(generics.ListAPIView):
```
Comment (m0nk-tnd) [inline notifications/views.py:6]: Еще можно создать эндпоинт для получения шаблонов уведомлений.

Comment (AnnaStepakova) [reply notifications/views.py:6]: fixed

File: notifications/views.py (hunk 1 above)
Comment (AnnaStepakova) [reply notifications/views.py]: fixed

File: notifications/migrations/0011_notification_receiver_user.py [hunk 3]
```diff
@@ -12,7 +12,7 @@ class Migration(migrations.Migration):
     operations = [
         migrations.AddField(
             model_name='notification',
-            name='receiver',
-            field=models.CharField(max_length=255, null=True),
+            name='receiver_user',
+            field=models.PositiveIntegerField(null=True),
```
Comment (m0nk-tnd) [inline notifications/migrations/0011_notification_receiver_user.py:16]: А это не может быть Foreign key?

File: notifications/models.py [hunk 4]
```diff
@@ -27,6 +27,8 @@ class Notification(models.Model):
                                  null=True)
     data = models.JSONField(default=dict, null=True)
     sent = models.BooleanField(default=False)
+    date_created = models.DateTimeField(auto_now_add=True, blank=True, null=True)
+    receiver_user = models.PositiveIntegerField(null=True)
```
Comment (m0nk-tnd) [inline notifications/models.py]: А это не может быть Foreign key?

Comment (AnnaStepakova) [reply notifications/models.py]: пофиксила

**Pull request merged**